pipelines/tasks/docker-compose.yml

```diff
@@ -0,0 +1,27 @@
+version: '2'
+services:
```

Comment:
Splitting into a separate compose file seems to be the recommended way to handle the situation where you have a top-level compose file defining services that make up the app's infrastructure, and another set of tasks which are expected to run to completion when they're invoked, but which rely on the top-level services in order to function. See here for more detail: https://github.com/docker/compose/issues/1896 .

Rereading some of that issue, however, it looks like there is a more idiomatic way to handle this by simply invoking `docker-compose` with multiple file parameters, so I think this could be refactored slightly to remove the `network` specification that's currently needed to link the two files together.